app/tests/archives_tests/test_views.py

```diff
@@ -228,21 +229,18 @@ def test_api_archive_api_detail_view(client):
 
 
 @pytest.mark.django_db
+@override_settings(task_eager_propagates=True, task_always_eager=True)
 def test_api_archive_item_allowed_sockets(
-    client, settings, django_capture_on_commit_callbacks
+    client, django_capture_on_commit_callbacks
```

Comment:
Minor style update on this test. I initially had the reserved part here, but thought to keep the updated style.